### PR TITLE
Exo2.6.0

### PR DESCRIPTION
**Pull request by sylvain-toptal**
It might not include all the latest and greatest features but I was able to get the new Downloader classes working and it's a good starting point.  Thanks for these bindings Martijn and take note that the time to update the bindings was paid for by WB.

### CODE REVIEW

File: ExoPlayer.SmoothStreaming/Additions/TrackKey.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// **********************************************************************
+// 
+//   TrackKey.cs
+//   
+//   This file is subject to the terms and conditions defined in
+//   file 'LICENSE.txt', which is part of this source code package.
+//   
+//   Copyright (c) 2017, Sylvain Gravel
```
Comment (martijn00) [inline ExoPlayer.SmoothStreaming/Additions/TrackKey.cs]: Can you remove the header?

**Comment by martijn00** [comment]
Are all the AAR files updated? And does the sample still work?

**Comment by sylvain-toptal** [comment]
There we go, sorry for the rushed out PR, I'm sometimes afraid I'll just forget to do something.
Yes all AARs have been updated and I played back a couple of different videos successfully.
What I didn't do however is dig through the warnings to see if anything was left out for some reason.  New extensions have also not been integrated so take this PR as an on-the-spot contribution and thanks again for saving us time.

**Comment by martijn00** [comment]
The warnings in bindings are way off sometimes, so don't always pay attention to it. I'll see if i can update the rest of the packages.

**Comment by sylvain-toptal** [comment]
Yes exactly, the warnings are usually useless noise which makes it hard to spot the real issues.  I usually try to spot the warning that mention that some XPath from metadata.xml can't be found.

Anyways, thanks for your time

**Pull request merged**